atomic_reactor/plugins/exit_store_metadata_in_osv3.py

```diff
@@ -228,7 +228,7 @@ def run(self):
         def parent_images_to_str(parent_images):
             results = {}
             for base_image_name, parent_image_name in parent_images.items():
-                results[base_image_name.to_str()] = parent_image_name.to_str()
```

Comment:
What if we check if `base_image_name` and `parent_image_name` are truthy before adding to results? I'd rather not have `None` in the annotations.

Reply:
None *shouldn't* get in there if the build succeeded to the point where it's storing metadata, but no harm in sanity checking.